numpy/core/src/npymath/npy_math_internal.h.src

```diff
@@ -795,3 +808,59 @@ npy_rshift@u@@c@(npy_@u@@type@ a, npy_@u@@type@ b)
 }
 /**end repeat1**/
 /**end repeat**/
+
+
+#define __popcnt32 __popcnt
+/**begin repeat
+ *
+ * #type  = ubyte, ushort, uint, ulong, ulonglong#
+ * #STYPE = BYTE,  SHORT,  INT,  LONG,  LONGLONG#
+ * #c     = hh,    h,      ,     l,     ll#
+ */
+#undef TO_BITS_LEN
+#if 0
+/**begin repeat1
+ * #len = 8, 16, 32, 64#
+ */
+#elif NPY_BITSOF_@STYPE@ == @len@
+    #define TO_BITS_LEN(X) X##@len@
+/**end repeat1**/
+#endif
+
+NPY_INPLACE npy_@type@
+npy_popcountu@c@(npy_@type@ a)
+{
+#if ((defined(__clang__) || defined(__GNUC__))) && NPY_BITSOF_@STYPE@ >= 32
+    return __builtin_popcount@c@(a);
+#elif defined(_MSC_VER) && NPY_BITSOF_@STYPE@ >= 16
+    return TO_BITS_LEN(__popcnt)(a);
```

Comment:
It seems like:
> In 32-bit mode, there are no 64-bit general-purpose registers, so 64-bit popcnt isn't supported.

Hence the [failure](https://dev.azure.com/numpy/numpy/_build/results?buildId=18780&view=logs&j=c2d14800-8d4a-5342-9575-ba481005464b&t=834033e0-67ab-57a9-dad0-2389f178e6e7&l=878). I shall fix it.